### ISSUE TITLE
ECK uninstall only works with kubectl 1.14+

### ISSUE THREAD

**Issue by pebrc** [elastic/cloud-on-k8s#1670]
We recommend using `kubectl delete elastic --all --all-namespace` but support for the `--all-namespaces` flag was only added in 1.14 afaik

**Comment by thbkrkr** [comment]
I verified, like you mention, the `--all-namespaces` flag is available only from the version 1.14 of kubectl.

I think about 2 ways to propose an alternative.

We could just document to execute the command in all namespaces.

In all namespaces where you installed Elasticseach, Kibana or APM Server, run:
```
kubectl delete elastic --all
```

Otherwise we could document a shell oneliner:
```
kubectl get namespaces --no-headers -o custom-columns=:metadata.name \
  | xargs -n1 kubectl delete elastic --all -n
```

I have a preference for the former which is simpler.

**Comment by pebrc** [comment]
I don't think I have a strong preference either way, figuring out all namespaces you have ECK managed resources running in might be tedious, that is why I like your shell magic :-)
